evergreen/compile-centos7-evergreen.sh

```diff
@@ -0,0 +1,13 @@
+ #!/bin/bash
+set -euo pipefail
+
+sudo yum install -y podman fuse-overlayfs conmon || sudo dnf install -y podman fuse-overlayfs conmon
```

Comment:
I'm sorry -- does this just "retry once no matter what happens, success or failure"?  Why would we do this?

Reply:
Yeah, this was copied from the SRE repo. I believe it's to retry on different linux distros, which is not necessary here. Removed.